pkg/storaged/btrfs/volume.jsx

```diff
@@ -49,6 +58,46 @@ const _ = cockpit.gettext;
  *        -> btrfs device
  *            -> block device
  */
+
+async function btrfs_delete(uuid, card) {
+    const volume = client.uuids_btrfs_volume[uuid];
+    const name = volume.data.label || uuid;
+
+    const usage = get_active_usage(client, volume.path, _("delete"), undefined, false, undefined, true);
+
+    if (usage.Blocking) {
+        dialog_open({
+            Title: cockpit.format(_("$0 is in use"), name),
```

Comment:
This is basically the original bug report in https://bugzilla.redhat.com/show_bug.cgi?id=2352385 . So this would hopefully not apply in the anaconda case?

Reply:
Correct. It should be impossible to get `usage.Blocking == true` here, actually. We _could_ make it an assertion.